heat/core/tests/test_arithmetics.py

```diff
@@ -16,25 +16,18 @@ def setUpClass(cls):
         cls.a_vector = ht.float32([2, 2])
```

Comment:
should the linter even run on the test files?

Reply:
It's Python code that is part of the project, so I'm arguing yes.